src/main/java/it/pn/frontend/e2e/section/destinatario/personaGiuridica/DelegatiImpresaSection.java

```diff
@@ -141,18 +142,30 @@ public void clickRevocaMenuButtonPG() {
     public void waitPopUpRevoca(String ragionSociale) {
         try {
             By titlePopUpBy = By.id("dialog-title");
-            this.getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(titlePopUpBy));
-            this.getWebDriverWait(30).until(ExpectedConditions.textToBe(titlePopUpBy, "Vuoi revocare la delega a " + ragionSociale + "?"));
+            this.getWebDriverWait(10).until(ExpectedConditions.visibilityOfElementLocated(titlePopUpBy));
+            this.getWebDriverWait(10).until(ExpectedConditions.textToBe(titlePopUpBy, "Vuoi revocare la delega a " + ragionSociale + "?"));
             logger.info("Il pop-up revoca si visualizza correttamente");
         } catch (TimeoutException e) {
             logger.error("Il pop-up revoca NON si visualizza correttamente con errore: " + e.getMessage());
             Assert.fail("Il pop-up revoca NON si visualizza correttamente con errore: " + e.getMessage());
         }
     }
 
+    public void esistenzaRevocaButton() {
+        try{
+            this.getWebDriverWait(10).withMessage("Bottone di revoca non visualizzato").until(ExpectedConditions.and(ExpectedConditions.visibilityOf(this.revocaMenuButton), ExpectedConditions.elementToBeClickable(this.revocaMenuButton)));
+            logger.info("Bottone revoca visualizzato e cliccabile");
+            Actions actions = new Actions(driver);
+            actions.moveByOffset(100, 200).click().perform();
```

Comment:
What is the purpose of this action?

Reply:
Click on a random point of the screen to close the menu opened when clicked on the 3 dot on the delegation